### PR TITLE
Add dir attribute to html tag

### LINKED ISSUE #611: Script (dir attribute) for Persian article
**Is your feature request related to a problem? Please describe.**
All of our Articles (News and Persian) default to using the default text direction, which is implied to be `dir="ltr"`.
For Persian articles, since we're using the arabic script, which is a right-to-left script, the `html` element for Persian articles should have a property `dir="rtl"`.  

There could be a mapping between 

**Describe the solution you'd like**

There is a quick solution - adding a function to handle mapping of service to `dir` and then adding it to the page with Helmet in our layout
```
<Helmet htmlAttributes={{ dir: dir(service) }} />
```
This would work since we currently do not have services with both `rtl` & `ltr` scripts at the top level.  

A more complete way of addressing this, which would be more accurate, would be to have a clear mapping of service to script and then note that anywhere we use the `arabic` script, we'll want `dir: 'rtl'`.

This would work by creating a React Context for Script, with the value for text direction. `dir`. 

- Create a `lib/config/scripts` directory with files for latin and arabic, plus an index.js file including the default. In each of them, there should be a `dir` attribute. (It should be structured like this one, for the services: https://github.com/BBC-News/simorgh/tree/latest/src/app/lib/config/services)
- Create a `ScriptContext` component (Similar to the ServiceContext component: https://github.com/BBC-News/simorgh/tree/latest/src/app/components/ServiceContext)  This component should use the above `lib/config/scripts`. 
- Update the `Metadata` component to take in the prop `dir` and add it to the `htmlAttributes`. https://github.com/BBC-News/simorgh/blob/latest/src/app/components/Metadata/index.jsx#L26
- Use the `ScriptContext` component in the Article container, surrounding the Metadata component, to pass the direction attribute in.

We note that later on, we could pull out the script mapping to dir attributes into a Psammead utility file. 
e.g. 
```
{
  arabic: {
    dir: 'rtl',
  },
  latin: {
    dir: 'ltr',
  },
  ...
}
```

**Additional context**
~Blocked by this issue: #610, creation of Persian Article routes.~ Now unblocked. 

**Approach should be discussed prior to picking up this  issue. Then should share the approach with the WS Front Page stream of  work.**

We now have the `@bbc/gel-foundations` package that exports scripts: https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/scripts.js

### PR DESCRIPTION
**Pull request by sareh**
Resolves #611

**Overall change:** Add `dir` to `<html>` tag - Is `ltr` for news and `rtl` for Persian.

**Code changes:**

- `dir` attribute from ServiceContext used in `ErrorMain` & value added to `htmlAttributes` via Helmet
- Same as above, but for the `Metadata` component. 
- Added cypress tests

Persian article with correct layout: http://localhost.bbc.com:7080/persian/articles/c4vlle3q337o
Screenshot_2019-06-20 پهپادی که برایتان قهوه می‌آورد
Persian error page with correct layout: http://localhost.bbc.com:7080/persian/articles/c0000000000o
Screenshot_2019-06-20 این صفحه یافت نشد - BBC News فارسی

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]
LGTM. Nice work 👍

File: cypress/integration/errorPageNews.js [hunk 1]
```diff
@@ -19,6 +20,10 @@ describe('Article Body Tests', () => {
     testResponseCode(`/news/articles/${config.assets.nonExistent}`, 404);
   });
 
+  it('should have the correct lang & dir attributes', () => {
+    hasHtmlLangDirAttributes({ lang: 'en_GB', dir: 'ltr' });
```
Comment (dr3) [inline cypress/integration/errorPageNews.js]: This will fail on test 🙈 unless we update mozart pre merge

Comment (sareh) [reply cypress/integration/errorPageNews.js]: Thanks for raising this. I often forget that the Mozart static error pages are used on Test and Live. I've limited this test for now, to be removed after this issue is complete: https://github.com/bbc/simorgh-infrastructure/issues/347

**Review by dr3 — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
CodeClimate is complaining about duplicated code on this one but I don't think it's having a good day today, so let's ignore that.

LGTM 👍

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**